content/docs/introducing-jsx.md

```diff
@@ -128,23 +135,25 @@ const element = (
 );
 ```
 
-### JSX Prevents Injection Attacks {#jsx-prevents-injection-attacks}
+### JSX Voorkomt Injectie Aanvallen {#jsx-prevents-injection-attacks}
 
-It is safe to embed user input in JSX:
+Het is veilig om invoer van gebruikers in JSX te gebruiken:
 
 ```js
 const title = response.potentiallyMaliciousInput;
-// This is safe:
+// Dit is veilig:
 const element = <h1>{title}</h1>;
 ```
 
-By default, React DOM [escapes](http://stackoverflow.com/questions/7381974/which-characters-need-to-be-escaped-on-html) any values embedded in JSX before rendering them. Thus it ensures that you can never inject anything that's not explicitly written in your application. Everything is converted to a string before being rendered. This helps prevent [XSS (cross-site-scripting)](https://en.wikipedia.org/wiki/Cross-site_scripting) attacks.
+React DOM [escapet <sup>[EN]</sup>](http://stackoverflow.com/questions/7381974/which-characters-need-to-be-escaped-on-html) alle waarden die ingevoegd worden in JSX voordat ze gerenderd worden. 
+Hiermee wordt verzekerd dat je nooit code kan injecteren die niet expliciet in je applicatie is geschreven. 
+Alles wordt omgezet naar een string voordat het gerenderd wordt. Dit helpt om [XSS (cross-site-scripting)](https://nl.wikipedia.org/wiki/Cross-site_scripting) aanvallen te voorkomen.
```

Comment:
English note

Reply:
No, this links to the Dutch wikipedia page for cross-site-scripting. I think with Wikipedia we should try to use the Dutch page if possible.

Reply:
oh oops, I looked at the original link, my bad 🤦‍♂️